### ISSUE TITLE
Should throw an error if two PathMaps share an identical remotepath or localpath

### ISSUE THREAD

**Issue by MitchLillie** [gwomacks/php-debug#209]
Steps to reproduce: 

0. Work on projects from two machines with different local directory structures, and share Atom settings between them for convenience.
1.  Enter PathMaps as the following:
```
    PathMaps: [
      "remotepath;localpath"
      "/sites/mysite/www;/Users/mitch/Documents/work/mysite"
      "/sites/mysite/www;/Users/mitch/Documents/home/mysite"
    ]
```
2. Attempt to set a breakpoint from the code stored in `home`

This would pertain to the opposite as well (a localpath points to two remotepaths).

Resolution:
* Reduce `PathMaps` in `remotePathToLocal` and check for identity before returning, showing a `Notification` to the user if duplicates are found.

Happy to work on a PR if this is something people are interested in.

**Comment by cchamplin** [comment]
It would be great if you could provide a PR to fix this.

**Comment by cchamplin** [comment]
Should be fixed in v0.3.0
